src/transformStateWithClones.js

```diff
@@ -66,6 +66,30 @@
  */
 function transformStateWithClones(state, transforms) {
   // write code here
+  const result = [];
+  const clone = { ...state };
```

Comment:
```suggestion
  const clones = [];
  const clone = { ...state };
```
Или transformedObject = { ...state }; transformedObjects  = [];